.docker/images/nginx/tests/src/FilesCacheTest.php

```diff
@@ -4,46 +4,43 @@
 
 use PHPUnit\Framework\TestCase;
 
-class FilesCacheTest extends TestCase
-{
+class FilesCacheTest extends TestCase {
+
```

Comment:
best practice for phpunit is to provide `setUp()` as a very first method in the class. Just moved to the top of the file with no changes.